src/averageAges.js

```diff
@@ -15,6 +15,19 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
+  const arrayOfMen = people.filter(person => {
+    return arguments.length < 2
+      ? person.sex === 'm'
+      : Math.ceil(person.died / 100) === century && person.sex === 'm';
+  });
+
+  const sumOfAges = arrayOfMen.reduce((acc, man) => {
+    return acc + (man.died - man.born);
+  }, 0);
+
+  const averageAge = sumOfAges / arrayOfMen.length;
+
+  return averageAge;
```

Comment:
you can return value right away
```suggestion
  return  sumOfAges / arrayOfMen.length;
```